### PR TITLE
adding search page UI

### PR DESCRIPTION
**Pull request by gargenal1**
I'm doing a PR just to say that I did one. but I'm not sure why I have to right now specifically.

### CODE REVIEW

File: public/css/searchstylesheet.css [hunk 1]
```diff
@@ -0,0 +1,39 @@
+body {
+    font-family: 'Roboto', Arial, sans-serif;
+  }
+  
+  /* .wrapper {
```
Comment (Theobasallaje) [inline public/css/searchstylesheet.css]: Please remove this commented out styles.
Thanks!

File: public/css/searchstylesheet.css [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+  .container {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+  }
+  
+  .searchBar {
+    width: 90%;
+    max-width: 600px;
+    /* display: flex;
```
Comment (Theobasallaje) [inline public/css/searchstylesheet.css]: These as well

File: public/pages/searchPage.html [hunk 3]
```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <title>Search Page</title>
+    <meta name="description" content="" />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <!--<link rel="stylesheet" href="" />-->
```
Comment (Theobasallaje) [inline public/pages/searchPage.html]: Please remove this comment, thanks!

**Review by Theobasallaje — CHANGES_REQUESTED** [review]
Just need to remove some comments, thanks!

File: public/css/searchstylesheet.css [hunk 4]
```diff
@@ -1,13 +1,6 @@
 body {
     font-family: 'Roboto', Arial, sans-serif;
   }
-
```
Comment (gargenal1) [inline public/css/searchstylesheet.css]: These comments have been deleted.

Comment (Theobasallaje) [reply public/css/searchstylesheet.css]: Awesome, thanks!

**Review by Theobasallaje — APPROVED** [review]

**Pull request merged**